crates/cuda-builder/src/lib.rs

```diff
@@ -18,12 +18,9 @@ pub struct CudaBuilder {
 impl Default for CudaBuilder {
     fn default() -> Self {
         let mut link_search_paths = Vec::new();
-        if let Ok(ld_path) = env::var("LD_LIBRARY_PATH") {
-            for path in ld_path.split(':') {
-                if !path.is_empty() {
-                    link_search_paths.push(path.to_string());
-                }
-            }
+        link_search_paths.push("/usr/local/cuda/lib64".to_string());
```

Comment:
can you undo this? not good to use `/usr/local/cuda` now that I realize we may have multiple cuda versions installed and explicitly don't want to go through the symlink